src/Illuminate/Redis/Connectors/PhpRedisConnector.php

```diff
@@ -91,6 +91,28 @@ protected function createClient(array $config)
             if (! empty($config['read_timeout'])) {
                 $client->setOption(Redis::OPT_READ_TIMEOUT, $config['read_timeout']);
             }
+            if (! empty($config['serializer'])) {
+                $serializers = [
+                    //Redis::SERIALIZER_NONE
+                    'none' => 0,
```

Comment:
Why not use `'none' => Redis::SERIALIZER_NONE`?

For me it looks cleaner.

Reply:
I assume due to the extension possibly not being installed?

Reply:
Also it depends on flags, which you used, when you compile ext-redis.
You can compile it w/o igbinary and constant wont be available

Reply:
Refactored to 'defined()' checks, so it will be more straight to understand in code